src/common/utils/kaltura-params.js

```diff
@@ -91,16 +91,18 @@ function addClientTag(source: Object) {
  * @private
  */
 function addKalturaParams(sources: Object, player: Player): void {
-  handleSessionId(player);
-  Object.keys(sources).forEach((key) => {
-    sources[key].forEach((source) => {
-      if (typeof source.url === 'string' && source.url.toLowerCase().indexOf(PLAY_MANIFEST) > -1) {
-        updateSessionIdInUrl(source, player.config.session.id);
-        addReferrer(source);
-        addClientTag(source);
-      }
+  if (!sources.dash || (sources.dash[0] && !sources.dash[0].localSource)) {
```

Comment:
let's make this a bit more generic, don't check for dash, just iterate over all sources and check for localSource:
```javascript
if ((typeof source.url === 'string') && 
    (source.url.toLowerCase().indexOf(PLAY_MANIFEST) > -1) &&
    !source.localSource
) {
```